specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/sourceControl.json

```diff
@@ -493,8 +494,8 @@
         },
         "securityToken": {
           "type": "string",
-          "maxLength": 1024,
-          "description": "Gets or sets the authorization token for the repo of the source control."
+          "description": "Gets or sets the authorization token for the repo of the source control.",
```

Comment:
Please avoid terminology like "Gets or sets" as this isn't idiomatic for all generated SDK languages. Just stick with the description. This also applies to the other descriptions below.

Reply:
Fixed.